extensions/test/algorithms/Jamfile

```diff
@@ -12,6 +12,7 @@ test-suite boost-geometry-extensions-algorithms
     :
 #    [ run dissolve.cpp ]
     [ run distance_info.cpp ]
+    [ run parse.cpp]
```

Comment:
This is no longer necessary since parse.cpp has already been added, in #802

Reply:
removed in new commit